src/site/xhp/ui-feedback.php

```diff
@@ -0,0 +1,104 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+namespace HHVM\UserDocumentation\ui;
+
+use namespace Facebook\XHP\Core as x;
+use type Facebook\XHP\HTML\{a, div, i, script};
+use type HHVM\UserDocumentation\{
+  github_issue_link,
+};
+
+final xhp class feedback extends x\element {
+
+  <<__Override>>
+  protected async function renderAsync(): Awaitable<x\node> {
+    $thumbs_up =
+      <a id="like-feedback" href="#"><i class="fa fa-solid fa-thumbs-up fa-lg"></i></a>;
+
+    $thumbs_down =
+      <a id="dislike-feedback" href="#"><i class="fa fa-solid fa-thumbs-down fa-lg"></i></a>;
+
+    $feedback_message =
+      <div id="feedback-prompt-message">Was This Page Useful?
+        {$thumbs_up}
+        {$thumbs_down}
+      </div>;
+
+    $good_feedback_message =
+      <div id="like-message">Thank You!</div>;
+
+    $bad_feedback_message =
+      <div id="dislike-message">Thank You. If you have feedback, you can <github_issue_link issueTitle={$this->getGithubIssueTitle()} issueBody={$this->getGithubIssueBody()}>File an Issue</github_issue_link> to help future readers!</div>;
```

Comment:
- 'File an Issue' feels like inconsistent captialization here
- perhaps reword to "If you're like to share feedback, please `[`file an issue`]` ?

Reply:
yeah, I agree. I was trying to make it stand out a bit but the anchor linking is probably good enough.

Reply:
Maybe another phrase other than "If you'd like to share feedback", because the reader is also sharing feedback. Something like "If you have any specific feedback..."?